src/common/app-paths/app-paths.test.ts

```diff
@@ -40,30 +40,32 @@ describe("app-paths", () => {
       recent: "some-recent",
       temp: "some-temp",
       videos: "some-videos",
-      userData: "some-irrelevant",
+      userData: "some-irrelevant-user-data",
     };
 
-    mainDi.override(
-      getElectronAppPathInjectable,
-      () =>
-        (key: PathName): string | null =>
-          defaultAppPathsStub[key],
-    );
-
-    mainDi.override(
-      setElectronAppPathInjectable,
-      () =>
-        (key: PathName, path: string): void => {
-          defaultAppPathsStub[key] = path;
-        },
-    );
-
-    mainDi.override(appNameInjectable, () => "some-app-name");
+    applicationBuilder.beforeSetups(({ mainDi }) => {
```

Comment:
Should `beforeSetups`'s name be changed because `setup`'s aren't a thing anymore?

Reply:
Changed to `beforeApplicationStart`